packages/sitecore-jss-react/src/index.ts

```diff
@@ -41,7 +41,7 @@ export {
 export { mediaApi } from '@sitecore-jss/sitecore-jss/media';
 export { ComponentFactory } from './components/sharedTypes';
 export { Placeholder, PlaceholderComponentProps } from './components/Placeholder';
-export { Image, ImageField } from './components/Image';
+export { Image, ImageField, getEEMarkup } from './components/Image';
```

Comment:
Do you think that `getEEMarkup` should be used outside of the `Image` component? I think it's internal only function

Reply:
@illiakovalenko We are using it in the `NextImage` component, so needs to be exported if we want to share unfortunately.

Reply:
@ambrauer So, it can be imported directly from the Image component? Instead of sharing it with user

Reply:
@illiakovalenko This function is exposed to use in the `sitecore-jss-nextjs` package. Even though a customer wouldn't likely use it, this is the only way to share package-to-package.